File: C++11/백준/Bronze/9635. Balloons Colors/Balloons Colors.cc
#include <iostream>
using namespace std;

int main(){
    int q;
    cin >> q;
    for (int i=0;i<q;i++){
        int n,a,b;
        cin >> n >> a >> b;
        int aa[n];
        for (int j=0;j<n;j++){
            cin >> aa[j];
        }
        if(aa[0]==a && aa[n-1]==b){
            cout << "BOTH" << endl;
        }
        else if(aa[0]==a && aa[n-1]!=b){
            cout << "EASY" << endl;
        }
        else if(aa[0]!=a && aa[n-1]==b){
            cout << "HARD" << endl;
        }
        else{
            cout << "OKAY" << endl;
        }
    }
}

File: Java/백준/Bronze/9635. Balloons Colors/Balloons Colors.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int q = in.nextInt();
        for (int i=0;i<q;i++){
            int n = in.nextInt();
            int a = in.nextInt();
            int b = in.nextInt();
            int[] aa = new int[n];
            for (int j=0;j<n;j++){
                aa[j]=in.nextInt();
            }
            if(aa[0]==a && aa[n-1]==b){
                System.out.println("BOTH");
            }
            else if(aa[0]==a && aa[n-1]!=b){
                System.out.println("EASY");
            }
            else if(aa[0]!=a && aa[n-1]==b){
                System.out.println("HARD");
            }
            else{
                System.out.println("OKAY");
            }
        }
    }
}